index.bs

```diff
@@ -136,22 +136,26 @@ interface DetectedFace {
 
 <pre class="idl">
 dictionary Landmark {
-  required Point2D location;
+  required FrozenArray&lt;Point2D> location;
   LandmarkType type;
 };
 </pre>
 
 <dl class="domintro">
   <dt><dfn dict-member for="Landmark"><code>location</code></dfn></dt>
   <dd>Location of a given landmark aligned to the image axes.</dd>
+  <dd>A point or a <a>sequence</a> of points of the detected landmark.</dd>
```

Comment:
This should be merged with the previous `<dd>` instead of
forming a new entry, i.e. every `<dt>` should have a unique
`<dd>` associated.